src/get-started/install/_ios-setup.md

```diff
@@ -19,11 +19,18 @@ To develop Flutter apps for iOS, you need a Mac with Xcode installed.
     If you need to use a different version, specify that path instead.
 
  1. Sign the Xcode license agreement.
-    To sign the SLA, either open Xcode once and confirm or run:
+    To sign the SLA, either:
 
-    ```terminal
-    $ sudo xcodebuild --license
-    ```
+    {: type="a"}
+    1. Open Xcode and confirm.
+
+    1. Open the Terminal and run:
+
+       ```terminal
+       $ sudo xcodebuild -license
+       ```
+
+       The single dash for the `license` option was introduced in Xcode 14.
```

Comment:
If you're going to say something was changed, you should generally explain what worked before. Otherwise, it will just cause extra reading or confusion on what to do before.

In this case, since the instructions say to install the latest version of xcode and that's all Apple makes easy to do, I think it should be fine to not include any note.

```suggestion
```

Reply:
Actually, I just looked through some related posts online and it seems the single dash syntax has (always?) been the supported syntax and wasn't just introduced in Xcode 14. So this note is definitely fine to remove.